soh/soh/Enhancements/randomizer/randomizerTypes.h

```diff
@@ -1040,7 +1040,7 @@ typedef struct ScrubIdentity {
     RandomizerCheck randomizerCheck;
     GetItemID getItemId;
     int32_t itemPrice;
-    bool isShuffled;
+    uint32_t isShuffled;
```

Comment:
Could this be like the smallest int type like uint_8 or something?

Reply:
This is more of a question than a suggestion, we should get this merged asap regardless

Reply:
Sorry I completely forgot about this. This should be possible, I'll change and test it when I'm in front of my PC again.

Reply:
uint_8 will take up as much space as char, which is 32 bits, so it doesn't actually help anything to use it

as for the bool problem, i think we should actually address the "bools don't work in c" issue in our codebase <https://stackoverflow.com/questions/1921539/using-boolean-values-in-c>

Reply:
> as for the bool problem, i think we should actually address the "bools don't work in c" issue in our codebase 

Are you suggesting the change for this PR simply be add `#include <stdbool.h>` at the top of `en_dns` or is it more complicated than that?

Reply:
i'm thinking we should add that include to one of the headers included by `z_en_dns.h` (`ultra64.h` or `global.h`)

Reply:
> i'm thinking we should add that include to one of the headers included by `z_en_dns.h` (`ultra64.h` or `global.h`)

From my reading this does sound like the more correct solution, but also sounds riskier. Maybe we roll with this as is for develop-zhora, and make the "correct" change on develop?

Reply:
sounds good, let's do the "switch to int as bool" fix for this pr on develop-zhora, and then make an issue for supporting bools and replacing our int as bool vars in develop

Reply:
```suggestion
    uint8_t isShuffled;
```

Reply:
https://github.com/HarbourMasters/Shipwright/issues/1680